src/scripts/create_ice_linguistic.py

```diff
@@ -0,0 +1,96 @@
+"""Create a linguistic acceptability dataset from the Icelandic Linguistic Benchmarks dataset."""
+
+from ast import literal_eval
+
+import pandas as pd
+import requests
+from datasets import Dataset, DatasetDict, Split
+from huggingface_hub import HfApi
+from requests.exceptions import HTTPError
+
+
+def main():
+    """Create a linguistic acceptability dataset from the Icelandic Linguistic Benchmarks dataset."""
+    dataset_url = "https://raw.githubusercontent.com/stofnun-arna-magnussonar/ice_linguistic_benchmarks/refs/heads/main/ice_benchmark_set.jsonl"
+
+    # Download the dataset and convert it to a dataframe
+    response = requests.get(dataset_url)
+    lines = [literal_eval(line) for line in response.text.split("\n") if line.strip()]
+    df = pd.DataFrame(lines)
+
+    train_df = prepare_dataframe(df=df)
+
+    # Make split from 382 samples. Train: 250, val: 50, test: 82
+    # Validation split
```

Comment:
Since we're making the splits, I would prioritise the test split over the others, for a more robust measure of test performance.

For instance, 256 test samples, 32 validation samples and 94 training samples.

Reply:
Note that the changelog should be updated as well.